crates/router/src/db/call_back_mapper.rs

```diff
@@ -0,0 +1,48 @@
+use error_stack::report;
+// use diesel_models::call_back_mapper;
+use router_env::{instrument, tracing};
+
+use super::Store;
+use crate::{
+    connection,
+    core::errors::{self, CustomResult},
+    types::storage,
+};
+
+#[async_trait::async_trait]
+pub trait CallBackMapperInterface {
```

Comment:
Can we please move the trait CallBackMapperInterface to hyperswitch_domain_models and have the domain models defined for CallBackMapper?

Reply:
this is a db interface, could you please tell me why db interface needs to be moved to domain model?